blocked-edges/4.18.0-rc.1-CMOSuddenStrictConfigValidation.yaml

```diff
@@ -1,5 +1,6 @@
 to: 4.18.0-rc.1
 from: ^4[.]18[.]0-ec[.][0-2][+].*$
+fixedIn: 4.18.0-rc.2
```

Comment:
From the previous commit https://github.com/openshift/cincinnati-graph-data/pull/6405/commits/14655100fabc8fc4f72480231518bed70d15e1f9:

> Bumping to EC3 will prevent the next RC (**likely** RC2)

Do we want to declare the `fixedIn` right now? I am unaware of how build process of RCs exactly works time-wise. Are we sure the RC2 won't have the upgrade paths from the affected versions? I am not sure, so I would personally declare `fixedIn` for RC2 after the RC2 is built.

Reply:
Yeah I don't mind dropping the commit to a separate PR and /holding it :+1:

Reply:
Thanks!